config/migration/migrate.go

```diff
@@ -0,0 +1,93 @@
+package migration
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"git_applet/config/migration/migration_types"
```

Comment:
wanna move the config types into config dir?

Reply:
They can't be, it'd be a circular dependency